### DISCUSSION TITLE
Using Prometheus Labels in Tooltips?

### DISCUSSION THREAD

**Discussion by Ryuzavi** [flant/grafana-statusmap#164] [Ideas]
Hey

Just discovered this plugin and it looks really promising.  I've played about with the panel and got it working fine but I was wondering if there was a way to enchance the tooltip to show more information for each value.  

Specifically I'm wondering if there's a way to use a label value in each tooltip like Grafana allows with labels.  So if I set the tooltip to `{{server}} up` it would show each value for the server label in the tooltip.

Is something like this possible?

Cheers,
R.

**Comment by diafour** [comment]
Hi! Did you mean the tooltip for multi value buckets? I think yes, it is possible to add options for the tooltip with templating.

**Comment by Ryuzavi** [comment]
Yes indeed.  That'd be great if added.  Cheers!

**Comment by diafour** [comment]
Did some investigation. This feature is not possible now with Prometheus datasource.

Prometheus response contains labels for each metric but prometheus datasource doesn't put labels in a result structure, so metric labels are not available in a panel plugin. :(

I will check other datasources.
